### ISSUE TITLE
Move to >= php7.0

### ISSUE THREAD

**Issue by rullzer** [nextcloud/previewgenerator#91]

**Comment by Schmuuu** [comment]
Hi  @rullzer 

I was getting warnings from the cronjob:
```
libpng warning: Interlace handling should be turned on when using png_read_image
libpng warning: Interlace handling should be turned on when using png_read_image
...
```
Now I saw this issue here. Might this be related and these messages just come due to missing php7.0+ support right now? Or should I open another issue for this?

I'm using php7.2.2 right now.

Nextcloud 13 (stable)
php7.2.2
ningx1.12.2
ArchLinux 4.14.17-lts

**Comment by stratege1401** [comment]
Hello there:

Nextcloud 13.0.4
PHP 7.0.27-0+deb9u1
    with Zend OPcache v7.0.27-0+deb9u1
Apache/2.4.25 (Debian)
zlib1g-dev amd64 1:1.2.8.dfsg-5 
libpng-dev amd64 1.6.28-1 
libpng-tools amd64 1.6.28-1 

Same for me.

`Scanning folder /.../files/PROJETNAS
libpng warning: Interlace handling should be turned on when using png_read_image
`
Something to do ??

**Comment by SchoolGuy** [comment]
I believe this is fixed with the tag 4.0.0

**Comment by st3iny** [comment]
Yeah, this has been done quite some time ago. BTW, rullzer is not responsible for the app anymore. I'm the new maintainer.
